src/logging/logging.vcxproj

```diff
@@ -66,7 +66,7 @@
       <RuntimeTypeInfo>true</RuntimeTypeInfo>
       <UseFullPaths>false</UseFullPaths>
       <WarningLevel>Level4</WarningLevel>
-      <PreprocessorDefinitions>WIN32;_WINDOWS;SPDLOG_COMPILED_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>WIN32;_WINDOWS;SPDLOG_COMPILED_LIB;SPDLOG_WCHAR_FILENAMES;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
is it necessary to add `SPDLOG_WCHAR_FILENAMES` here? I'd assume importing `spdlog.props` would add it.

Reply:
You are right. Removed it here

Reply:
Not clear why this has been marked as resolved, should `SPDLOG_WCHAR_FILENAMES` be removed or not?